src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -322,11 +323,19 @@ private static SyntaxList<UsingDirectiveSyntax> GetUsings(SyntaxNode node)
             // some * "(" cases
             if (currentToken.Kind() == SyntaxKind.OpenParenToken)
             {
+                if (previousToken.Kind() == SyntaxKind.NewKeyword)
+                {
+                    // new (
+                    //
+                    // 'new' could be a modifier in a declaration, or this could an implicit obj or new-constraint.
+                    var spaces = previousToken.Parent?.Kind() is SyntaxKind.ImplicitArrayCreationExpression or SyntaxKind.ConstructorConstraint ? 0 : 1;
```

Comment:
It turns out the `ImplicitArrayCreationExpression` check isn't necessary here (that's what I was investigating just now), but I couldn't figure out why tests pass without it... I'm good to keep it there.

Reply:
The block at line 316 above can be removed.